### ISSUE TITLE
Why the loss while training dataset of coco128 was always 4.5+ and never decreased and convergented, please and thanks!

### ISSUE THREAD

**Issue by Logan-KY** [WongKinYiu/yolov9#279]
![image](https://github.com/WongKinYiu/yolov9/assets/78793122/09c78975-404a-4fbb-b48d-418157eae860)
![image](https://github.com/WongKinYiu/yolov9/assets/78793122/76f3f8e8-e60c-422d-aab2-b8f2fd5227a2)
![image](https://github.com/WongKinYiu/yolov9/assets/78793122/d5bbcec3-c7ab-4d0f-a53d-4ca789410420)
![image](https://github.com/WongKinYiu/yolov9/assets/78793122/88021b33-7ad6-4d47-bd41-e4166d843760)

Following all the steps, the coco.yaml file was modified to use yolov9c.pt for train_dual.
but the loss of it is always 4.5+ and didn't decrease.
the train result looks normal, but the val result is so chaotic.(the effect of val_batch_pred is chaotic, but it is normal in val_batch_labels)
Please help me out thanks!

**Comment by Logan-KY** [comment]
![image](https://github.com/WongKinYiu/yolov9/assets/78793122/60d683ed-30e9-4c70-b449-3240d847a8c3)
![image](https://github.com/WongKinYiu/yolov9/assets/78793122/b5c4895e-6b13-49f3-95cc-6092c79ab782)
I'm so confused.
